### ISSUE TITLE
PSReadLine doesn't work properly with Windows Terminal's background image

### ISSUE THREAD

**Issue by lanyizi** [PowerShell/PSReadLine#1842]
## Environment

```none
PS version: 7.0.3
PSReadline version: 2.1.0-beta2
os: 10.0.18362.1074 (WinBuild.160101.0800)
PS file version: 7.0.3.0
HostName: ConsoleHost (Windows Terminal)
BufferWidth: 120
BufferHeight: 30
```

## Steps to reproduce & Expected behavior & Actual behavior
(From https://github.com/PowerShell/PowerShell/issues/13720)

I think the screenshot below should be self-explanatory, but let me explain more precisely.
Windows Terminal allows us to set a background image, then it will replace the area occupied by the original background color with the background image.
But when I'm typing commands that contain spaces, the background color of text returned to be the original background color, instead of being "transparent" (or more preciesly, instead of displaying the content of background image).
It only occurs when I'm typing commands, looks like this problem won't be triggered when I'm writing a string literal, even if it contains spaces, as shown in the screenshot.
![image](https://user-images.githubusercontent.com/2989995/94719751-da071c00-0353-11eb-9962-14cec928e12b.png)

If I unload PSReadLine module, the issue won't occur anymore.
![image](https://user-images.githubusercontent.com/2989995/94745830-bfe13400-037b-11eb-8fd2-4da7fc0e04ca.png)

**Comment by DHowett** [comment]
this is a duplicate of #830, and only happens if you're using the "dotnet global" tool version of PowerShell _and_ haven't updated PSReadline.

[Here's a bit about it in the Terminal docs](https://docs.microsoft.com/en-us/windows/terminal/troubleshooting#my-colors-look-strange-there-are-black-bars-on-my-screen)

**Comment by lanyizi** [comment]
@DHowett Yes, I have "dotnet global" tool version of PowerShell core, but I think my PSReadLine is already up-date?
My PSReadline version is 2.1.0-beta2, and I still have this issue..

**Comment by vexx32** [comment]
Hmm, looking over the releases, it looks like that fix was made after the beta release and released as 2.0.3; I don't see another beta release after that, so it looks like it hasn't been rolled into a beta branch just yet.

**Comment by lanyizi** [comment]
@vexx32 looks like that's it, I uninstalled the pre-release version of PSReadLine and issue has been fixed.
